blog/2023-06-22-openfeature-adoption-story.md

```diff
@@ -0,0 +1,130 @@
+---
+slug: 'openfeature-adoption-story'
+title: 'OpenFeature Adoption Story'
+date: 2023-06-22
+authors: [liran2000]
+description: "OpenFeature is an open standard for feature-flagging. This is an adoption story."
+tags: [client, sdk, adoption]
+draft: false
+---
+
+Our team recently needed to use one of the largest feature management cloud services.
+This service has a documented SDK with usage guidelines.
+Thinking about how to define the exposed interfaces, while keeping it simple and generic, I started doing some research, and encountered [OpenFeature](https://openfeature.dev).
+Gladly, the mentioned feature management cloud service has already created an OpenFeature provider.
+Let me explain why and how it was adopted.
+
+<!--truncate-->
+
+## Why Feature Management was needed
+
+Our product needed a solution for feature flags:
+* providing a gradual release mechanism and a simple way to define target audiences such as target group or split release
+* Using feature flags for dynamic configuration values which can be changed in runtime dynamically without needing for a new release deployment.
+* Ability for A/B testing - can be done via feature for specific region, specific users, etc.
+
+## Why OpenFeature was chosen
+
+While keeping it simple and generic, I had a few wonders:
+* How to name the method
+* How to do provide information for error handling
+* What response structure to define
+* How to define properties passed to the methods
+
+OpenFeature solved me all these questions with its standard. When using a standard, I have good reasons why it is defined like this, as the standard supports it.
```

Comment:
What do you think about changing line 34 to the below? Does it capture the essence of what you're trying to say?

Because OpenFeature is an open standard, we felt confident in the way things are defined, and it answered all of the above questions for us.